clang/lib/Sema/SemaDecl.cpp

```diff
@@ -17940,6 +17940,12 @@ Decl *Sema::getObjCDeclContext() const {
 }
 
 Sema::FunctionEmissionStatus Sema::getEmissionStatus(FunctionDecl *FD) {
+  // Due to SYCL functions are template we check if they have appropriate
```

Comment:
Doesn't seem to be a full sentence. Word missing?

Reply:
Can't see what is missing in "Due to SYCL functions are template we check if they have appropriate attribute prior to checking if it is a template"